src/argilla/client/feedback/dataset/local/mixins.py

```diff
@@ -311,6 +317,9 @@ def push_to_argilla(
             guidelines=self.guidelines,
             allow_extra_metadata=self.allow_extra_metadata,
         )
+        rich.print(f"[bold green]✓ Dataset succesfully pushed to Argilla[/bold green]")
+        print(remote_feedback_dataset)
```

Comment:
Is this `print` here intended?

Reply:
Yes, to return the repr of the RemoteDataset.